<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelToolClass</class>
 <widget class="QMainWindow" name="LabelToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LabelTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:white;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="MyLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>800</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>800</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>点2</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>点3</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>标注：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>点1：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>点4</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_x3"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_x2"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_x1"/>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_x4"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_y1"/>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="lineEdit_y2"/>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="lineEdit_y3"/>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="lineEdit_y4"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>横坐标</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>纵坐标</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_clear">
          <property name="text">
           <string>重置标记</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>读数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_numbers"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_save">
          <property name="text">
           <string>保存(Enter)</string>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_rotate">
          <property name="text">
           <string>旋转90度(Ctrl+C)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_delete">
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="pushButton_larger">
            <property name="text">
             <string>放大</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_smaller">
            <property name="text">
             <string>缩小</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>当前文件夹为：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_dirPath">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>文件夹中图片总数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_imgNum">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>已标记数目：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_labelNum">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>剩余图像数目：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_leftImgNum">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_next">
          <property name="text">
           <string>下一张(Ctrl+X)</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_next_labeled_image">
          <property name="text">
           <string>查看下一张已标注图片（Ctrl+Z）</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+Z</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Dir"/>
    <addaction name="actionOpen_label"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>已标注数据查看</string>
    </property>
    <addaction name="action_label_dir"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionLabel">
   <property name="text">
    <string>label</string>
   </property>
  </action>
  <action name="actionOpen_Dir">
   <property name="text">
    <string>open Dir</string>
   </property>
  </action>
  <action name="actionOpen_label">
   <property name="text">
    <string>open label</string>
   </property>
  </action>
  <action name="action_label_dir">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">MyLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="labeltool.qrc"/>
 </resources>
 <connections/>
</ui>
